ci/citest/acceptance-test/tests/local_image_test.go

```diff
@@ -3,20 +3,46 @@
 package tests
 
 import (
+	"fmt"
+	"path/filepath"
 	"strings"
 	"testing"
 	"time"
 )
 
 func TestLocalImage(t *testing.T) {
 
+	stdout, _, err := RunSsh(scheduler_ip, fmt.Sprintf("[ -f /var/www/html/images/centos/7/amd64/meta.tar.xz ] && echo meta.tar.xz found || echo meta.tar.xz not found"))
+
+        if err != nil {
+                t.Error(err)
+        }
+
+        t.Log(stdout.String())
+
 	// Use custom lxc-template.
-	stdout, _ := RunCmdAndReportFail(t, "openvdc", "run", "centos/7/lxc", `{"lxc_template":{"template":"openvdc"}, "node_groups":["linuxbr"]}`)
+	stdout, _ = RunCmdAndReportFail(t, "openvdc", "run", "https://raw.githubusercontent.com/axsh/openvdc/template-misc/templates/centos/7/lxc2.json", `{"node_groups":["linuxbr"]}`)
```

Comment:
when this branch gets removed after merging, the test will fail. lxc2.json should be bundled as test fixtures.

Please refer:
- https://github.com/axsh/openvdc/tree/template-misc/ci/citest/acceptance-test/tests/fixtures
- https://github.com/axsh/openvdc/blob/template-misc/ci/citest/acceptance-test/tests/cmd_template_test.go#L11-L17